### PR TITLE
Added Code Path config information to Scope v0.2.2

### PR DESCRIPTION
**Pull request by drodriguezhdez**
Added Code Path config information to Scope v0.2.2

### CODE REVIEW

File: docs/java-configuration.md [hunk 1]
```diff
@@ -43,6 +43,31 @@ scope:
...
 
+## Adding code path information
+
+You can add code path information to certain spans to show the executed lines given a certain test in a certain commit.
+
+Apart from the activation/deactivation of the code path flag, it is needed to configure which packages will be needed to be tracked by the Scope Java Agent.
+
+You just need to indicate the base package, and every sub-package, starting from that point, will be included as observable package.
+
+### Using Environment Variables
```
Comment (fermayo) [inline docs/java-configuration.md]: Why the env vars don't match the YAML configuration file? Shouldn't they match?

Comment (drodriguezhdez) [reply docs/java-configuration.md]: Probably it was long time ago when the variable was introduce. In fact, the variable is called `SCOPE_CODE_COVERAGE`, and IMO, it should be named `SCOPE_CODE_PATH` or something similar.

Nevertheless, I think the name `SCOPE_INSTRUMENTATION_TEST_FRAMEWORKS_CODE_COVERAGE` is so long. Maybe we should name it as `SCOPE_CODE_PATH`, moving the property in the YAML in the correct position.

Let's sync with other agents @nacho-undefinedlabs @tonyredondo, what you think?

Comment (drodriguezhdez) [reply docs/java-configuration.md]: Updated with `SCOPE_CODE_PATH` and its correspondent YAML prop.

File: docs/java-configuration.md [hunk 2]
```diff
@@ -43,6 +43,29 @@ scope:
...
+SCOPE_CODE_PATH=true
+SCOPE_CODE_PATH_BASE_PACKAGES="foo.bar.xyz, bar.foo"
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  code_path: true
+  code_path_base_packages: "foo.bar.xyz, bar.foo"
```
Comment (fermayo) [inline docs/java-configuration.md]: Maybe we could do:
```yaml
scope:
  code_path:
    enabled: true
    base_packages: "foo.bar.xyz, bar.foo"
```

So we could add code path configuration in the future? cc @undefinedlabs/sdks

**Pull request merged**